domains/fallen.json

```diff
@@ -0,0 +1,13 @@
+{
+  "description": "Personal site and portfolio.",
+  "repo": "https://github.com/ThingsOfFallen/Website",
+  "owner": {
+    "username": "TheFallenSpirit",
+    "email": "thefallenspirit@outlook.com",
+    "twitter": "@FallenCodes"
+  },
+  "record": {
+    "A": "198.58.110.227",
```

Comment:
Also, A records must be an array

Reply:
Updated A record to an array, removed AAAA record.